lambdas/indexer/app.py

```diff
@@ -310,3 +311,13 @@ def consolidate(self, others: List['DocumentTally']) -> 'DocumentTally':
         assert all(self.entity == other.entity for other in others)
         return replace(self, num_contributions=sum((other.num_contributions for other in others),
                                                    self.num_contributions))
+
+
+@app.schedule('rate(5 minutes)')
```

Comment:
We should tie this to the lambda timeout so we don't have more than one concurrent execution of this lambda. I'm thinking 110% of the timeout.